packages/protocol/contracts-0.8/common/EpochManager.sol

```diff
@@ -264,8 +275,17 @@ contract EpochManager is
       epochProcessing.totalRewardsCarbonFund
     );
     // run elections
-    elected = election.electValidatorAccounts();
-    _epochProcessing.status = EpochProcessStatus.NotStarted;
+
+    address[] memory _newlyElected = election.electValidatorAccounts();
+
+    electedAccounts = _newlyElected;
+
+    _setElectedSigners(_newlyElected);
+
+    EpochProcessState memory _epochProcessingEmpty;
+    epochProcessing = _epochProcessingEmpty;
```

Comment:
Do we need to zero this out here ?

Reply:
This was a suggestion from @martinvol. I don't have any issues with zeroing it out at the end of the epoch process.

Reply:
leaving as it is now